Berichte funktionieren mit englischer Sprache nicht

* Browsersprache auf englisch stellen
* in Zeiterfassung anmelden und `Reports` aufrufen

der Graph links ist sichtbar und rechts, je nach Datenstand, gar nichts oder irgendwann hört der Inhalt einfach auf.

**erwartetes Verhalten**

Die Berichte Seite wird mit allen Informationen korrekt angezeigt.

Info:

```log
org.thymeleaf.exceptions.TemplateInputException: An error happened during template parsing (template: "class path resource [templates/reports/user-report.html]")
(...)
Caused by: java.lang.IllegalArgumentException: Cannot format given Object as a Date
(...)
Caused by: org.attoparser.ParseException: Error during execution of processor 'org.thymeleaf.standard.processor.StandardTextTagProcessor' (template: "reports/user-report-week" - line 354, col 27)
```

--- a/src/main/resources/messages_en.properties
+++ b/src/main/resources/messages_en.properties
@@ -177,7 +177,7 @@ report.detail.week.section.title=Calendar week {2}: {0,date,short} to {1,date,sh
 report.detail.no-entries=No times have been recorded for this day yet.
 report.detail.day.head={0}, {1} ({2, choice, 0#{2} Hours|1#1 Hour|1.01#{2} Hours})
 report.detail.day.comment.label=Comment:
-report.detail.time={0,time,short} to {1,time,short}
+report.detail.time={0} to {1}
 
 usermanagement.heading=Persons
 usermanagement.teaser-text=Information: Only persons who have logged in at least once in the time recording are displayed here.

